api/controller/MortgageRateController.py

```diff
@@ -0,0 +1,116 @@
+from flask import Response, request
+from api.models.mortgage_rate import MortgageRate
+from mongoengine.errors import FieldDoesNotExist, NotUniqueError, DoesNotExist, ValidationError, InvalidQueryError
+from api.helpers.errors import APINotUniqueError, APISchemaError, APIDoesNotExistError
+from re import sub
+import pry
```

Comment:
When we are ready to merge to `main`, I'm putting this out there to remind ourselves to take out the `pry`